src/components/projectRegister/RegisterHeader.tsx

```
@@ -1,67 +1,311 @@
+        // 전체 완료 항목 (필수 + 선택)
+        let totalCompleted = requiredCompleted
+
+        // 이미지는 선택사항이지만 있으면 전체 카운팅에 추가
+        if (formData.images && formData.images.length > 0) {
+          totalCompleted += 1
+        }
+
+        return {
+          completed: totalCompleted,
+          total: requiredFields.length + 1, // 이미지 포함하여 총 8개
+          requiredCompleted,
```

Biome (1.9.4)

[error] Prefer for...of instead of forEach.

forEach may lead to performance issues when working with large arrays. When combined with functions like filter or map, this causes multiple iterations over the same type.

(lint/complexity/noForEach)